### PR TITLE
MIA-170,168,251,263, and 249

### PR DESCRIPTION
**Pull request by kennethedwards-bixal**
Ticket attempts to close above referenced tickets that will
-remove waiver
-include eslint/prettier
-update node version to 16
-remove ENOME error in pipeline
-script will return non zero code
-include unit test

TEST: checkout stage branch, pull down, and run `node buildwaiverdata.js`

### CODE REVIEW

**Comment by scottqueen-bixal** [comment]
work should be merged into `develop` first, https://github.com/GSA/made-in-america-data/blob/develop/CONTRIBUTING.md#branches

otherwise `develop` will be out of sync which could break others' work who are following the contribution guidelines

**Comment by kennethedwards-bixal** [comment]
It was merged into develop first.
- I made branch called hotfix/nodeversion, then merged that into develop.
Christian approved it and then merged it into stage.
- then I merged your edits into stage.
Now I'm ready to merge into main.

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```
Comment (jefffortune) [inline .vscode/launch.json:2]: JSON doesn't allow inline comments.

**Review by jefffortune — CHANGES_REQUESTED** [review]
There are two conflicts
- waivers-data.json
- waivers.csv
Can you please fix these conflicts, thank you.

File: .vscode/launch.json (hunk 1 above)
Comment (kennethedwards-bixal) [reply .vscode/launch.json:2]: Conflicts resolved

**Pull request merged**